### ISSUE TITLE
No DPI

### ISSUE THREAD

**Issue by TrendMend** [inkcut/inkcut#174]
This ties in with #173 . There is no dpi setting for devices... This is required (unfortunately) for my craft cutter, even though it should have nothing to do with the scale, but with the resolution. Is there a remedy for this?

**Comment by frmdstryr** [comment]
You can change the output scaling under devce setup. Inkscape now defaults to 96 DPI so you'll need to set the scale to 400/96.

**Comment by TrendMend** [comment]
That is what I did for at the moment (I did 400/1051 because inkscape used to use it at the time of the firmware release) and it worked well enough, but you can only go two decimal places which is not optimal for things needing more precision...

**Comment by DamienBurkeJJ** [comment]
Agreed, minimum of 4 decimal places would be better. My cutter, a Pixmax 720, wants somewhere between 0.91 and 0.92 for perfect scaling.

**Comment by frmdstryr** [comment]
Just increased the decimal places to 6 https://github.com/inkcut/inkcut/commit/47aa6b99092cb7f574a28d49ee783373c3338464 and also pushed out a release.

Run `pip3 install --upgrade inkcut`

**Comment by jarney** [comment]
Hello.  I realize this is coming out of nowhere, but I encountered a problem with Inkcut not cutting correct dimensions on my plotter.  The issue is also discussed in [this thread](https://inkcut.org/t/inch-mm/15/).  I am offering a pull-request to address this by creating an explicit DPI setting and also allowing Inkcut to detect whether it is an Inkscape document and set the DPI appropriately based on the version of Inkscape (90 DPI for versions below 0.92 and 96DPI thereafter).  

I realize this issue is really old, but perhaps this PR will be of use to someone.

**Comment by jarney** [comment]
I believe that PR #420 closes this issue.
